app/index.html

```diff
@@ -4,10 +4,32 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/reset.css">
     <link rel="stylesheet" href="styles/main.css">
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <header>
+        <div class="center align-left" id="logo">
+            <span id="logo-text" class="primary-text-color">Welcome</span>
```

Comment:
- [ ]  It's required that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring that. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of `UX` convention make the navigation in the website way more pleasant and intuitive.

Reply:
Enclosed the logo in <a> tag.